docs/sources/solacemqtt.md

```diff
@@ -2,6 +2,10 @@
 
 This event source is meant to be used as a Container Source with a Knative cluster to consume messages from [Solace pubsub+](https://solace.com/) and send them to a Knative service/function.
 
+## Prerequisite(s)
+
+- Solace
+
```

Comment:
Not part of our public facing docs, can be removed.

Reply:
@odacremolbap are you meaning to delete the whole file?

Reply:
yes delete